src/test/java/org/apache/commons/io/input/buffer/CircularBufferInputStreamTest.java

```diff
@@ -70,6 +70,8 @@ public void testRandomRead() throws Exception {
 				throw new IllegalStateException("Unexpected random choice value");
 			}
 		}
+		bais.close();
+		cbis.close();
 	}
```

Comment:
This should be the subject of a separate PR

Reply:
@sebbASF agreed.